sagemaker-datawrangler/readme.md

```diff
@@ -1 +1,41 @@
+![Amazon SageMaker Data Wrangler](https://github.com/aws/amazon-sagemaker-examples/raw/main/_static/sagemaker-banner.png)
```

Comment:
Not sure why this change is here... seems to be the same file as what's in the repo now.